src/api/api.js

```diff
@@ -1,9 +1,23 @@
 import firebase from "firebase";
 
-export const getCurrentUser = (userId) => firebase.database().ref(`/users/${ userId }`).once('value')
+export const getUser = (userId) => firebase.database().ref(`/users/${ userId }`)
```

Comment:
rename this function to the setUser and pass arguments for update in current function